lib/schema.js

```diff
@@ -30,5 +30,5 @@ module.exports = Joi.object({
     styleSrc: Joi.array().items(Joi.string()).single().default(['self'])
         .when('generateNonces', { is: [false, 'script'], then: Joi.array().items(Joi.string().valid('strict-dynamic').forbidden()) }),
     workerSrc: Joi.array().items(Joi.string()).single().default(['self']),
-    generateNonces: Joi.alternatives().try([Joi.boolean(), Joi.string().valid('script', 'style')]).default(true)
+    generateNonces: Joi.alternatives().try(Joi.boolean(), Joi.string().valid('script', 'style')).default(true)
```

Comment:
Joi migration.  They transitioned to using spread args instead of an array of args.